docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   arlocal:
-    image: textury/arlocal:v1.1.35
+    image: textury/arlocal:v1.1.66
```

Comment:
older versions of arlocal provide timestamps to the milliseconds which throws off our evaluation - newer ones use seconds